neurovault/apps/statmaps/views.py

```diff
@@ -174,6 +180,71 @@ def edit_collection(request, cid=None):
     return render(request, "statmaps/edit_collection.html.haml", context)
 
 
+def choice_datasources(model):
+    statmap_field_obj = functools.partial(image_metadata.get_field_by_name,
+                                          model)
+    pick_second_item = functools.partial(map, lambda x: x[1])
+
+    fixed_fields = list(model.get_fixed_fields())
+
+    field_choices = ((f, statmap_field_obj(f).choices) for f in fixed_fields)
+
+    fields_with_choices = (t for t in field_choices if t[1])
+
+    return dict((field_name, pick_second_item(choices))
+                for field_name, choices in fields_with_choices)
+
+
+def cognitive_atlas_task_datasource(request):
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [map/filter on lambda could be replaced by comprehension](https://www.codacy.com/app/krzysztof-gorgolewski/NeuroVault_NeuroVault/file/825092872/issues/source?bid=2767031&fileBranchId=2767194#l199)